tests/modules/nf-core/nanoplot/nextflow.config

```diff
@@ -3,8 +3,3 @@ process {
     publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
 
 }
-
-// Conda takes too long to resolve the dependencies
-conda {
-    useMamba = true
-}
```

Comment:
Similarly, this change is already on master: https://github.com/nf-core/modules/blob/master/tests/modules/nf-core/nanoplot/nextflow.config 
GitHub shouldn't be showing it